examples/browser/test/cpp/cpp-change-build-config.ui-spec.ts

```diff
@@ -0,0 +1,159 @@
+/********************************************************************************
+ * Copyright (C) 2018 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import * as utils from '../utils';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as cp from 'child_process';
+import { MainPage } from '../main-page/main-page';
+import { TopPanel } from '../top-panel/top-panel';
+import { LeftPanel } from '../left-panel/left-panel';
+import { BottomPanel } from '../bottom-panel/bottom-panel';
+
+let mainPage: MainPage;
+let topPanel: TopPanel;
+let leftPanel: LeftPanel;
+let bottomPanel: BottomPanel;
+
+before(() => {
+    const driver = browser;
+
+    driver.url('/');
+    driver.localStorage('DELETE');
+    driver.refresh();
+
+    mainPage = new MainPage(driver);
+    topPanel = new TopPanel(driver);
+    leftPanel = new LeftPanel(driver);
+    bottomPanel = new BottomPanel(driver);
+
+    mainPage.waitForStartup();
+});
+
+/**
+ * Prepare some test files in the workspace.
+ */
+function prepareWorkspace(): string {
+    // Put our stuff in a cpp/ subdirectory, because we share the workspace with other tests...
+    const rootDir = path.join(utils.getWorkspaceRoot());
+    const cppRootDir = path.join(rootDir, 'cpp');
+
+    fs.mkdirSync(cppRootDir);
+    fs.writeFileSync(path.join(cppRootDir, 'paul.cpp'), `
```

Comment:
How about `source.cpp`...?

Reply:
clangd is hard-coded to only work with files called `paul.cpp`.

Reply:
How convenient!